### ISSUE TITLE
[TESTING]: Telephone component needs an axe-check in unit tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/vets-design-system-documentation#492]
# Bug Report

- [ ] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened
Our [Telephone component](https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/Telephone/Telephone.jsx) doesn't have an axe-check to evaluate it for accessibility in the unit test suite. 

## Acceptance criteria
- [ ] Axe check is added to unit test ( https://github.com/department-of-veterans-affairs/component-library/blob/master/src/components/Telephone/Telephone.unit.spec.jsx )
- [ ] 0 axe-core violations reported

**Comment by bkjohnson** [comment]
The telephone web component [has various `axeCheck()`s that it is passing](https://github.com/department-of-veterans-affairs/component-library/blob/3fc3b7d33d9cfe5ac51a54d7aed0b445c26cfe05/packages/web-components/src/components/va-telephone/test/va-telephone.e2e.ts#L13-L43). We won't be adding any more unit tests to the `<Telephone>` React component since the web component replaces it.
